src/libraries/System.Linq.Parallel/src/System/Linq/ParallelEnumerable.cs

```diff
@@ -5821,9 +5810,9 @@ public static TSource SingleOrDefault<TSource>(this ParallelQuery<TSource> sourc
         /// <exception cref="System.ArgumentNullException">
         /// <paramref name="source"/> is a null reference (Nothing in Visual Basic).
         /// </exception>
-        public static ParallelQuery<TSource> DefaultIfEmpty<TSource>(this ParallelQuery<TSource> source)
+        public static ParallelQuery<TSource?> DefaultIfEmpty<TSource>(this ParallelQuery<TSource> source)
         {
-            return DefaultIfEmpty<TSource>(source, default!);
+            return DefaultIfEmpty<TSource>(source, default!)!;
```

Comment:
Why is this last `!` needed?

Reply:
Because `DefaultIfEmpty` now returns an enumerable of `T?`